llvm/test/Verifier/AMDGPU/intrinsic-amdgpu-init-exec-from-input.ll

```diff
@@ -0,0 +1,27 @@
+; RUN: not llvm-as -disable-output 2>&1 %s | FileCheck %s
+
+; Function Attrs: convergent nocallback nofree nounwind willreturn
+declare void @llvm.amdgcn.init.exec.from.input(i32, i32 immarg) #0
+attributes #0 = { convergent nocallback nofree nounwind willreturn }
```

Comment:
Does the attribute matter as well?

FWIW I'd prefer to keep the test as small as possible.

Reply:
No, I removed it, thanks. I guess all the attributes are irrelevant since the verifier is rejecting the module anyway.